### DISCUSSION TITLE
Creality Dynamic Logos for Fluidd

### DISCUSSION THREAD

**Discussion by OxygenGS** [Guilouz/Creality-Helper-Script-Wiki#422] [Issues]
This does not work in the new script. i installed it, seems successful in the SSH window, but then in the dashboard you cannot select the themes 9they are not listed)

**Comment by Guilouz** [comment]
It's working fine:

<img width="1247" alt="Capture d’écran 2024-03-11 à 13 03 18" src="https://github.com/Guilouz/Creality-K1-Series/assets/12702322/c664f805-951d-4f19-ad58-9f309707527e">

Reset your web browser cache.

**Comment by OxygenGS** [comment]
That did work, thank you! And thank you for these awesome and easy to use tools here man, its amazing! Keep up the great work!
